### PR TITLE
Int 365 add tests

### PR DESCRIPTION
**Pull request by Ragge-dev**
Solves:
https://annotell.atlassian.net/browse/INT-352 (Tests for dangling inputs)
https://annotell.atlassian.net/browse/INT-362 (Tests for Annotation-types)
https://annotell.atlassian.net/browse/INT-363 (Tests for create/publish batch)

I think I (by mistake) merged the get_inputs with input_uuid PR into this branch, so there is some extra in this one. Otherwise it is just changing the examples, adding some examples and adding tests.

### CODE REVIEW

File: annotell-input-api/annotell/input_api/resources/input/input.py [hunk 1]
```diff
@@ -41,14 +42,17 @@ def get_inputs(
         :param batch: Batch (identifier) to filter
         :param invalidated: Returns invalidated inputs if True, otherwise valid inputs
         :param external_id: External ID to filter input on
+        : param input_uuids: A UUID to filter inputs on
```
Comment (fredrikjacobson) [inline annotell-input-api/annotell/input_api/resources/input/input.py:45]: whitespace 👾

File: annotell-input-api/examples/cameras.py [hunk 2]
```diff
@@ -39,6 +45,7 @@ def run(client: IAC.InputApiClient, project: str, dryrun: bool = True) -> Option
 
     # Project - Available via `client.project.get_projects()`
     project = "Project-identifier"
-    run(client, project)
-
+    # Annotation Types - Available via `client.project.get_annotation_types(project)`
+    annotation_types = ["Annotation-type"]
```
Comment (fredrikjacobson) [inline annotell-input-api/examples/cameras.py:49]: lets stick to lower case for these.

File: annotell-input-api/tests/project_test.py [hunk 3]
```diff
@@ -30,9 +31,28 @@ def test_get_project_batches(self, client: IAC.InputApiClient):
 
         assert all([cameras_project == batch.project for batch in project_batches])
 
-    def test_get_annotation_types(self, client: IAC.InputApiClient):
+    def test_get_project_annotation_types(self, client: IAC.InputApiClient):
         projects = client.project.get_projects()
-        cameras_project = self.filter_cameras_project(projects)[0].project
-        annotation_types = client.project.get_annotation_types(cameras_project)
+        print([project.project for project in projects])
```
Comment (fredrikjacobson) [inline annotell-input-api/tests/project_test.py:36]: print can be removed? 🚧

**Review by fredrikjacobson — APPROVED** [review]
Nice addition to our tests. 
Have we looked at updating our local and test env. to reflect these new requirements?

**Comment by Ragge-dev** [comment]
> Nice addition to our tests.
> Have we looked at updating our local and test env. to reflect these new requirements?

Yes, the update to our local test env is in a separate PR, and the staging env is already updated

**Pull request merged**